config/m4/compiler.m4

```diff
@@ -440,40 +440,23 @@ ADD_COMPILER_FLAGS_IF_SUPPORTED([[-Wno-missing-field-initializers],
                                  [-Wno-multichar],
                                  [-Wno-deprecated-declarations],
                                  [-Winvalid-pch],
-                                 [-Wvariadic-macros]de],
+                                 [-Wvariadic-macros]],
                                 [AC_LANG_SOURCE([[int main(int argc, char **argv){return 0;}]])])
 
-
 #
 # Set C++ optimization/debug flags to be the same as for C
 #
 BASE_CXXFLAGS="$BASE_CFLAGS"
+AC_SUBST([BASE_CXXFLAGS], [$BASE_CXXFLAGS])
```

Comment:
can't you just set `AC_SUBST([BASE_CXXFLAGS])` ?

Reply:
didn't get your comment

Reply:
just shorter syntax

Reply:
why moved here instead of line 476?

Reply:
flags below are not supported by C++ compiler & produces command line error

Reply:
but BASE_CXXFLAGS="$BASE_CFLAGS" is done before the below anyway?
and ADD_COMPILER_FLAGS_IF_SUPPORTED is only affecting BASE_CFLAGS..

Reply:
ok, fixed